CGBEmu/source/Player.cpp

```diff
@@ -1,24 +1,28 @@
+#include "Player.h"
 #include <iostream>
-class Player
+
+Player::Player()
+	: width(0)
```

Comment:
In the constructor, which is called when an instance of a Player object is created, we need to set the initial values of the member variables. It is important to initialize the variables in the same order as they are defined in the header. 9i.e. width first and then height.